packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -223,6 +223,7 @@ class FinalizationRound(OnlyKeeperSendsRound):
     round_id = "finalization"
     allowed_tx_type = FinalizationTxPayload.transaction_type
     payload_attribute = "tx_data"
+    synchronized_data_class = SynchronizedData
```

Comment:
This highlights an issue with the current design. The base class sets this attribute to `BaseSynchronizedData`.

```
Error: An error occurred during act of behaviour valory/register_termination_abci:0.1.0/RegisterTerminationAbciAppConsensusBehaviour:
Traceback (most recent call last):

  File "/tmp/tmp0d8tdc4f/agent_00003/vendor/valory/skills/abstract_round_abci/base.py", line 636, in get
    raise ValueError(

ValueError: 'most_voted_keeper_address' field is not set for this period [0] and no default value was provided.
```

https://github.com/valory-xyz/open-autonomy/blob/05fb30f861114c2f1fb9b8731c748356596b7cb1/packages/valory/skills/abstract_round_abci/base.py#L1384-L1396

Reply:
This might be required elsewhere in our other repos.